src/main/java/com/revaturelabs/ask/user/UserServiceImpl.java

```diff
@@ -156,5 +168,66 @@ public User updateTags(User user) {
 
     return updatedUser;
   }
+
+  @Override
+  public User updateUserInfo(User user) {
+    // TODO Auto-generated method stub
+    return null;
+  }
+
+  
+  /**
+   * Function that will convert a multipart file to a file and upload it
+   * to an S3 bucket.
+   * 
+   * On the s3, it will save it to profilePictures/USERNAME/PICTURENAME
+   */
+  @Override
+  public String uploadProfilePicture(MultipartFile image, String username) {
+    
+    String key = "";
+    
+    //AWS connections configured
+    AWSCredentials credentials = new BasicAWSCredentials(
+        System.getenv("s3_access_key"), 
+        System.getenv("s3_secret_key")
+      );
+    
+    AmazonS3 s3client = AmazonS3ClientBuilder
+        .standard()
+        .withCredentials(new AWSStaticCredentialsProvider(credentials))
+        .withRegion(Regions.US_EAST_1)
+        .build();
+    
+    //Conversion block from multipartfile to image, then upload to s3
+    try {
+      byte[] bytes = image.getBytes();
+      if (bytes == null) {
+        throw new ImageConflictException("Invalid image");
```

Comment:
I'm not sure ImageConflictException is reasonable here.. what's the actual problem that occurs if getBytes() returns null?  We want an Exception that describes that